plugins/woocommerce/includes/admin/class-wc-admin-importers.php

```diff
@@ -32,6 +32,7 @@ public function __construct() {
 		add_action( 'admin_menu', array( $this, 'add_to_menus' ) );
 		add_action( 'admin_init', array( $this, 'register_importers' ) );
 		add_action( 'admin_head', array( $this, 'hide_from_menus' ) );
+		add_action( 'admin_head', array( $this, 'set_active_submenu_for_product_importer' ) );
```

Comment:
Nit. Maybe use a "menu_highlight" name here? The "product_importer" is assumed by the context. Same for exporter.